x/cvm/simulation/sim_utils.go

```diff
@@ -103,6 +104,38 @@ func DeployContract(caller sim.Account, contractCode string, contractAbi string,
 	return msg, deployResponse.Result, nil
 }
 
+func RandomReasonableFees(r *rand.Rand, ctx sdk.Context, spendableCoins sdk.Coins) (sdk.Coins, error) {
+	if spendableCoins.Empty() {
+		return nil, nil
+	}
+
+	perm := r.Perm(len(spendableCoins))
+	var randCoin sdk.Coin
+	for _, index := range perm {
+		randCoin = spendableCoins[index]
+		if !randCoin.Amount.IsZero() {
+			break
+		}
+	}
+
+	if randCoin.Amount.IsZero() {
+		return nil, fmt.Errorf("no coins found for random fees")
+	}
+	//To limit the fee not exceeding ninth of remaining balances.
+	//Without this limitation, it's easy to run out of accounts balance
+	//  given so many simulate operations introduced by shentu modules.
+	amt, err := sim.RandPositiveInt(r, randCoin.Amount.Add(sdk.NewInt(8)).Quo(sdk.NewInt(9)))
```

Comment:
The result of this operation is always 0?
`sdk.NewInt(8)).Quo(sdk.NewInt(9))`

Reply:
the 8 is added on `randCoin` before being divided by 9.

Reply:
Oh, I didn't notice the extra parentheses...